src/device/test/Dual_Call.cu

```diff
@@ -32,6 +32,8 @@ __host__ __device__ unsigned long long sub_conserve_calc_num_src_particles(
 
   ntot = 0;
 
+  Insist_device(part_per_e>0, "Use the Insist_device function");
+
```

Comment:
`clang-format` style check failure on this line.  Otherwise, LGTM.

Reply:
Sorry, should be fixed now. Thanks!